tests/seq-properties.hs

```diff
@@ -560,6 +562,23 @@ prop_sort :: Seq OrdA -> Bool
 prop_sort xs =
     toList' (sort xs) ~= Data.List.sort (toList xs)
 
+data UnstableOrd = UnstableOrd
+    { ordKey :: OrdA
+    , _val :: A
+    } deriving (Eq,Show)
```

Comment:
No, definitely don't derive `Eq`!!! It must match the `Ord` instance! You'll need to `fmap` an `unignore` function over each of the results in the tests.